scripts/analytics/user_summary.py

```diff
@@ -19,7 +19,7 @@
 def count_user_logs(user):
     logs = NodeLog.find(Q('user', 'eq', user._id))
     length = logs.count()
-    if length > 0:
+    if length >= LOG_THRESHOLD:
```

Comment:
No, just `==`. If it's greater than, subtracting one won't make a difference to the function that uses it, so we only need to do the check under one very specific instance.